ANSWERS.md

```diff
@@ -1,7 +1,14 @@
 <!-- Answers to the Short Answer Essay Questions go here -->
 
 1. In Mocha, what are the differences between `before`, `after`, `beforeEach`, and `afterEach`? When do they run? What are they used for?
+  In Mocha there are 4 lifecycle hooks available:
+  * `before` is run at the beginning of the test suite before any it blocks
+  * `after` is run at the end of the test suite after all describe blocks
+  * `beforeEach`is run at the beginning of each it block
+  * `afterEach` is run at the end of each it block
 
 2. What is the point of Test Driven Development? What do you personally think about this approach?

```

Comment:
I 100% agree with this. I think it goes along the lines of planning too much ahead, but I guess it does have it's benefits.